### PR TITLE
Added a parameter to set query execution settings

### PR DESCRIPTION
**Pull request by aleksvagachev**
##### SUMMARY
In continuation #63 implemented the installation of any number of parameters

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: plugins/modules/clickhouse_client.py [hunk 1]
```diff
@@ -44,6 +45,15 @@
         through the I(execute) argument.
     type: dict
     default: {}
+
+  set_settings:
+    description:
+      - The list of settings that need to be set before executing the request
+        Sets the passed parameters in the session before executing the request
```
Comment (Andersson007) [inline plugins/modules/clickhouse_client.py:52]:
```suggestion
      - The list of settings that need to be set in the session before executing the request.
```

File: plugins/modules/clickhouse_client.py [hunk 2]
```diff
@@ -64,6 +74,10 @@
   register: result
   community.clickhouse.clickhouse_client:
     execute: CREATE TABLE test_table_1 (x String) ENGINE = Memory
+    set_settings:
+      - flatten_nested = 0
+      - short_circuit_function_evaluation = 'disable'
```
Comment (Andersson007) [inline plugins/modules/clickhouse_client.py:79]: wouldn't a dict work better here? e.g.
```
    set_settings:
      flatten_nested: 0
      short_circuit_function_evaluation: disable
```
I'm just asking:)

**Comment by aleksvagachev** [comment]
@Andersson007 I corrected it for the dictionary

**Review by Andersson007 — APPROVED** [review]
@aleksvagachev the implementation looks very good now, thanks for the contribution!

**Pull request merged**